samples/cpp/benchmark_app/README.md

```diff
@@ -221,51 +221,45 @@ Below are fragments of sample output static and dynamic networks:
 
 * For static network:
    ```
-   [Step 10/11] Measuring performance (Start inference asynchronously, 4 inference requests using 4 streams for CPU, limits: 60000 ms duration)
-   [ INFO ] BENCHMARK IS IN INFERENCE ONLY MODE.
-   [ INFO ] Input blobs will be filled once before performance measurements.
-   [ INFO ] First inference took 26.26 ms
-   Progress: [................... ]  99% done
-
+   [Step 10/11] Measuring performance (Start inference asynchronously, 10 inference requests, limits: 10 iterations)
+   [ INFO ] Benchmarking in inference only mode (inputs filling are not included in measurement loop).
+   [ INFO ] First inference took 895.31 ms
    [Step 11/11] Dumping statistics report
-   [ INFO ] Count:      6640 iterations
-   [ INFO ] Duration:   60039.70 ms
+   [ INFO ] Count:        10 iterations
+   [ INFO ] Duration:     1425.72 ms
    [ INFO ] Latency:
-   [ INFO ]        Median:  35.36 ms
-   [ INFO ]        Avg:    36.12 ms
-   [ INFO ]        Min:    18.55 ms
-   [ INFO ]        Max:    88.96 ms
-   [ INFO ] Throughput: 110.59 FPS
+   [ INFO ]    Median:     1130.53 ms
+   [ INFO ]    Average:    1153.94 ms
+   [ INFO ]    Min:        1092.91 ms
+   [ INFO ]    Max:        1424.79 ms
+   [ INFO ] Throughput:   7.01 FPS
    ```
 
 * For dynamic network:
    ```
-   [Step 10/11] Measuring performance (Start inference asynchronously, 4 inference requests using 4 streams for CPU, limits: 60000 ms duration)
-   [ INFO ] BENCHMARK IS IN FULL MODE.
-   [ INFO ] Inputs setup stage will be included in performance measurements.
-   [ INFO ] First inference took 26.80 ms
-   Progress: [................... ]  99% done
-
+   [Step 10/11] Measuring performance (Start inference asynchronously, 10 inference requests, limits: 10 iterations)
+   [ INFO ] Benchmarking in full mode (inputs filling are included in measurement loop).
+   [ INFO ] First inference took 122.00 ms
    [Step 11/11] Dumping statistics report
-   [ INFO ] Count:      5199 iterations
-   [ INFO ] Duration:   60043.34 ms
+   [ INFO ] Count:        10 iterations
+   [ INFO ] Duration:     173.63 ms
    [ INFO ] Latency:
-   [ INFO ]        Median:  41.58 ms
-   [ INFO ]        Avg:    46.07 ms
-   [ INFO ]        Min:    8.44 ms
-   [ INFO ]        Max:    115.65 ms
+   [ INFO ]    Median:     153.86 ms
+   [ INFO ]    Average:    154.69 ms
+   [ INFO ]    Min:        124.62 ms
+   [ INFO ]    Max:        171.94 ms
    [ INFO ] Latency for each data shape group:
-   [ INFO ] 1. data : [1, 3, 224, 224]
-   [ INFO ]        Median:  38.37 ms
-   [ INFO ]        Avg:    30.29 ms
-   [ INFO ]        Min:    8.44 ms
-   [ INFO ]        Max:    61.30 ms
-   [ INFO ] 2. data : [1, 3, 448, 448]
-   [ INFO ]        Median:  68.21 ms
-   [ INFO ]        Avg:    61.85 ms
-   [ INFO ]        Min:    29.58 ms
-   [ INFO ]        Max:    115.65 ms
-   [ INFO ] Throughput: 86.59 FPS
+   [ INFO ] 1. input_1: {1, 32, 32, 3}
```

Comment:
1. Which model is used for output example? It seems it is not `asl-recognition-0004` which is mentioned above
2. Could you please look at this PR https://github.com/openvinotoolkit/openvino/pull/13555 and keep one

Reply:
Updated the docs with F16 asl-recognition-0004 model & added the changes from the mentioned PR! What do you mean by 'keep one'?

Reply:
> Updated the docs with F16 asl-recognition-0004 model & added the changes from the mentioned PR! What do you mean by 'keep one'?

Both PR replaces the same part.